mandatory/2-function-creation.js

```diff
@@ -25,7 +46,12 @@ Write a function that:
 - removes any forward slashes (/) in each string
 - makes all strings all lowercase
 */
-function tidyUpStrings(arrayOfStrings) {}
+function tidyUpStrings(arrayOfStrings) {
```

Comment:
Nicely done.
Using the lessons learnt during our session on chaining, we now know that we can also either :

1. Chain all the .map commands together, or
2. Chain all the string functions (trim, replace and toLowerCase ) together

Well done!